ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioHttpInAutoConfig.java

```diff
@@ -15,8 +15,9 @@
 
 @Configuration
 public class LdioHttpInAutoConfig {
+	public final static String componentName = "be.vlaanderen.informatievlaanderen.ldes.ldio.LdioHttpIn";
```

Comment:
Why are you using a different name here (componentName instead of NAME)? I also expect that sonar will tell you to swap static and final.